### PR TITLE
Make the user interface context safe for the initial setup

### PR DESCRIPTION
**Pull request by poncovka**
The context needs to be safe to use for the initial setup, so provide
a very limited specific API that will work in both projects.

### CODE REVIEW

**Review by jkonecny12 — APPROVED** [review]
Looks good to me. Sorry for still not having KS tests for initial-setup working. :(

File: pyanaconda/ui/context.py [hunk 1]
```diff
@@ -28,27 +26,26 @@ class UserInterfaceContext(object):
     of the user interface. The goal is to replace all
     these objects with this one.
 
+    WARNING: The context has to be usable by the initial setup!
```
Comment (M4rtinK) [inline pyanaconda/ui/context.py:29]: I think we can formulate this better:
```

WARNING: The UserInterfaceContext class is also used by the Initial Setup tool. 
         Please keep that in mind when doing any API breaking changes.
```

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks a lot! :)
Yeah, this should really make things much more robust & also looks cleaner. :)

File: pyanaconda/ui/context.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/ui/context.py]: Fixed.

**Comment by poncovka** [comment]
Updated.

**Pull request merged**